.changeset/chilly-emus-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/app': patch
+---
+
+App - provide better checks for the existence of the internal heartbeat cache to help debug issues in Opera.
```

Comment:
Nit: I wouldn't narrow this to Opera, maybe just "more robust check to cover more cases of empty heartbeat data" or something, worded better. Can refer to the Github issue in the release notes to point out the immediate use case.